front-end/src/environments/environment.ts

```diff
@@ -1,18 +1,18 @@
 // This file can be replaced during build by using the `fileReplacements` array.
-// `ng build ---prod` replaces `environment.ts` with `environment.prod.ts`.
+// `ng build --configuration=cloud.gov.dev` replaces `environment.ts` with `environment.cloud.gov.dev.ts`.
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false,
-  name: 'development',
-  apiUrl: 'http://localhost/api/v1',
-  appTitle: 'FECfile',
-  validateSuccess: 'All required fields have passed validation.',
-  awsRegion: 'us-east-1',
-  awsIdentityPoolId: 'us-east-1:f0f414b2-8e9f-4488-9cc1-34a5918a1a1d',
-  ACCESS_KEY: "_process.env.ACCESS_KEY", //TODO: fix this. Hacked to get Angular 8 upgrade working
-  SECRET_KEY: "_process.env.SECRET_KEY", //TODO: fix this. Hacked to get Angular 8 upgrade working
-  dcfConverterApiUrl: 'https://dev-efile-api.efdev.fec.gov/dcf_converter/v1'
+    production: false,
+    name: 'development',
+    apiUrl: 'https://fecfile-web-api-dev.app.cloud.gov/api/v1',
+    appTitle: 'FECfile',
+    validateSuccess: 'All required fields have passed validation.',
+    awsRegion: 'us-east-1',
```

Comment:
I don't think we need the AWS config variables, let's remove them

Reply:
Removed

Reply:
Just did a test and "ng server" generates errors is we remove awsRegion or awsIdentityPoolId so I am going to leave them in place.